src/lib/tenant/tenant.ts

```diff
@@ -53,6 +53,13 @@ export default class Tenant {
     return this._token;
   }
 
+  public get blockExplorer(): string {
```

Comment:
We had similar thoughts on how to handle block explorer links. I moved this stuff out into utils but the logic is very similar.